packages/js/src/lib/initialize.ts

```diff
@@ -26,9 +26,9 @@ const logger = Logger.getInstance();
 let isInitialized = false;
 
 const setDebugLevel = (c: TJsConfigInput): void => {
-  if (c.debug) {
-    logger.debug(`Setting log level to debug`);
+  if (c.debug || window.location.search.includes("formbricksDebug=1")) {
```

Comment:
Can we also use `formbricksDebug=true` instead of `formbricksDebug=1` for consistency or is using `1` instead of `true` more common in URL parameters?
We currently use `debug: true` in the JS SDK and also already have URL parameters like `preview=true` for link surveys.

Reply:
ok 👌